packages/dm-core-plugins/src/table/TableRow/TableRow.tsx

```diff
@@ -75,8 +75,11 @@ export function TableRow(props: TTableRow) {
       <Table.Row key={item.key}>
         {config.functionality.openAsExpandable && (
           <Table.Cell>
-            <Tooltip title={item.expanded ? 'Minimize' : 'Expand'}>
+            <Tooltip title={item.expanded ? 'Minimize' : 'Collapse'}>
```

Comment:
This seems wrong. It should be Expand, right?

Reply:
Oh, I changed the wrong one 🙈 Maybe it is time for summer holidays